Makefile

```diff
@@ -134,7 +134,7 @@ test-verbose: ginkgo-set tidy-vendor
 	@go test $(GO_BUILD_FLAGS) -v ./... --race --bench=. -cover --count=1
 
 format:
-	go fmt ./...
+	gofmt -l -w pkg/
```

Comment:
what about those in `cmd`?

Reply:
humm, good catch! `cmd` must be included

btw, the motivation to remove `./...` is because it was formatting the `vendors/`